atest/robot/standard_libraries/reserved.robot

```diff
@@ -13,3 +13,9 @@ Continue
 
 Reserved in User Keyword
     Check testcase    Reserved In User Keyword
+
+Else should be capitalized
+    Check testcase    Else should be capitalized
+
+Else If should be capitalized
+    Check testcase    Else If should be capitalized
```

Comment:
Newline still missing at the end of this file. You can see a special symbol when you look at the changes in GitHub UI.

The reason we want newlines at the end is that otherwise diffs later can be a bit messed up. It's easy to avoid missing them by configuring your IDE to add them automatically.